### PR TITLE
Add research_in_progress scope to BoxRequest

### LINKED ISSUE #406: Add research_in_progress scope to BoxRequest
(this page should load: http://localhost:9000/box_requests?utf8=%E2%9C%93&filter_by=research_in_progress, accessed when viewing Status, and filtering on ResearchInProgress

### PR DESCRIPTION
**Pull request by joyrm**
Resolves #406 

### Description

Added research_in progress scope to BoxRequest
### Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not
  work as expected)
- This change requires a documentation update
- Documentation update

### CODE REVIEW

File: app/models/box_request.rb [hunk 1]
```diff
@@ -18,13 +18,14 @@ class BoxRequest < ApplicationRecord
   delegate :name, to: :reviewed_by, prefix: :reviewer, allow_nil: true
 
   scope :requested, ->(){ where(reviewed_by_id: nil) }
-  scope :designed, ->(){ joins(:box).where("boxes.aasm_state = ?", "desgined") }
+  scope :designed, ->(){ joins(:box).where("boxes.aasm_state = ?", "designed") }
```
Comment (h-m-m) [inline app/models/box_request.rb:21]: Nice typo catch!

**Comment by h-m-m** [comment]
Hi, @joyrm — it was nice meeting you yesterday!

Do you think this change would benefit from a test? I could see an argument either way. What do you think?

**Comment by joyrm** [comment]
> Hi, @joyrm — it was nice meeting you yesterday!
> 
> Do you think this change would benefit from a test? I could see an argument either way. What do you think?

I don't know that it would be that helpful but I went with the "no one else tested scopes" argument.

**Comment by zachhweed** [comment]
Should we add the new state to the `aasm` code block as well?

**Comment by maebeale** [comment]
@zachhweed hello! what file/where are you suggesting we add it to the aasm code block? `research_in_progress` should already exist as a state, but we hadn't yet added a scope in order to filter box_requests accordingly, so i think what you're suggesting is already handled. happy to learn of an oversight tho! please share.

**Comment by zachhweed** [comment]
@maebeale it was a misunderstanding on my part. thanks :)

**Review by maebeale — APPROVED** [review]

**Pull request merged**